lib/workers/branch/schedule.ts

```diff
@@ -36,9 +39,24 @@ export function hasValidSchedule(
   }
   // check if any of the schedules fail to parse
   const hasFailedSchedules = schedule.some((scheduleText) => {
+    const parsedCron = parseCron(scheduleText);
+    if (parsedCron !== undefined) {
+      if (
+        parsedCron.minutes.length !== 60 ||
+        scheduleText.indexOf(minutesChar) !== 0
```

Comment:
In what scenario is the second check executed?

Reply:
Still missing response here

Reply:
@olegkrivtsov still missing your answer here

Reply:
Wouldn't that be caught by the config validation earlier?

Reply:
That's a good point, I'll remove the second condition. Just wanted to do validation here as well.